startup/81-pe-callbacks.py

```diff
@@ -165,6 +165,18 @@ def dark_plan_old(cam, dark_frame_cache, obsolete_secs, shutter):
 from event_model import RunRouter
 from suitcase.tiff_series import Serializer
 
+
+def pilatus_serializer_factory(name, doc):
+    serializer = Serializer(
+        '/nsls2/data/qas-new/legacy/processed/{year}/{cycle}/{PROPOSAL}XRD'.format(**doc),
+        file_prefix = (
+            '{start[sample_name]}-'
+            '{start[exposure_time]:.1f}s-'
+            '{start[scan_id]}-'
+        )
+    )
```

Comment:
Looks like it's another candidate for a saver caproto IOC.

Reply:
Yep. The PE and PIL detectors both use these serializers. Will need to be moved from profile for Data security.